grails-app/services/org/broadinstitute/orsp/IssueService.groovy

```diff
@@ -489,4 +489,15 @@ class IssueService implements UserInfo {
         IssueType.valueOfName(issue?.getType())?.prefix?.toLowerCase()
     }
 
+    /**
+     * Project can have PM ACTOR CREATOR.
+     * CG only have CREATOR.
+     * @param extraProperties
+     * @return
```

Comment:
nit: either remove or fill it in. This is a package protected method, probably doesn't need it. I don't feel strongly about javadoc in general. The only exception is for non-trivial methods that do some substantial work. This one is pretty self explanatory.